0014-runtime-signing-tx-with-hardware-wallet.md

```diff
@@ -122,7 +124,7 @@ The first three items in the derivation path are hardened.
 
 | Field   | Type      | Content               | Note                     |
 | ------- | --------- | --------------------- | ------------------------ |
-| PK      | byte (32) | Public Key            |                          |
+| PK      | byte (33) | Compressed Public Key |                          |
```

Comment:
Public keys should be compressed in almost all cases, so maybe we can skip this or add parentheses, e.g. Public Key,  Public Key (compressed), Public Key (compressed point), (Compressed) Public Key.